debian/changelog

```diff
@@ -1,3 +1,12 @@
+flatpak (1.4.99-1endless1) eos; urgency=medium
```

Comment:
I've asked @wjt  this the other day on slack and the conclusion was that there was no "standard" here - I've seem our packages using `0` and others using `1`, so not sure, but can certainly update if you prefer.

Reply:
Oh, I thought we standardised on `0`. If there’s no standard, then there’s no point in respinning this branch just for the sake of 1 arbitrary bit. Ignore me :-)

Reply:
It's the other way around. If `1.4.99` is not present in Debian, our version should be `1.4.99-0endlessX` for any value of `X` (AFAIK). I believe this field is parsed by merge-o-matic to work out what the common ancestor version is.

In this case it's academic because `1.4.99` will never exist in Debian.

Reply:
> It's the other way around.

Then the [documentation on the wiki](https://phabricator.endlessm.com/w/howto/rebase_debian_packages/#ask-someone-to-review-th) might need to be updated. Having re-read it, its wording doesn’t actually quite apply in this situation, but is confusing anyway.

Reply:
Should I updated it to 1.4.99-0endless1 then? or not worth it?

Reply:
I don't think it matters in this case because 1.4.99 is never really going to exist